borealis/usrp_drivers/usrp_driver.cpp

```diff
@@ -25,13 +25,13 @@
 #include <cmath>
 #include <tuple>
 #include <sys/mman.h>
-#include "utils/driver_options/driveroptions.hpp"
-#include "usrp_drivers/usrp.hpp"
-#include "utils/protobuf/driverpacket.pb.h"
-#include "utils/protobuf/rxsamplesmetadata.pb.h"
-#include "utils/shared_memory/shared_memory.hpp"
-#include "utils/shared_macros/shared_macros.hpp"
-#include "utils/zmq_borealis_helpers/zmq_borealis_helpers.hpp"
+#include "utils/driveroptions.hpp"
+#include "usrp.hpp"
+#include "borealis/utils/protobuf/driverpacket.pb.h"
+#include "borealis/utils/protobuf/rxsamplesmetadata.pb.h"
+#include "utils/shared_memory.hpp"
```

Comment:
The directory structure of the build directory is like this:

```
build/
    release/
        bin/
            (executables go here)
        testing/
            (C++ files from testing directory here)
        borealis/
            usrp_drivers/
                usrp.cpp
                usrp_driver
                usrp_driver.cpp
                usrp_driver.o
                usrp.hpp
                usrp.o
                utils/
                    driveroptions.hpp
                    libutils.a
                    options.hpp
                    shared_macros.hpp
                    shared_memory.hpp
                    zmq_borealis_helpers.hpp
                    (plus .cpp and .o files for all of these)
            utils/
                protobuf/
                    libmessages.a
                    driverpacket.pb.h
                    rxsamplesmetadata.pb.h
                    (plus .py, .cc, .o, and .proto versions of these)
```

So, only the protobuf library needs to start with borealis/, since all the others are either in the same directory or in a subdirectory of same directory. Protobuf is the exception, so it is getting a more explicit path.

Reply:
I guess more importantly, there are two `/utils/` directories - one in `borealis/` and another in `borealis/usrp_drivers/`